src/Notification/Model/Notification.php

```diff
@@ -39,6 +50,7 @@ class Notification extends Model
         'title'    => 'string',
         'content'  => 'string',
```

Comment:
Deze cast kan niet, dit was altijd dus al fout want hier moet ook een array in kunnen, je kan deze cast weghalen OF we kunnen ervoor kiezen dat we er hier in de backend altijd al een array van maken en dat die logica weg kan uit de frontend.